savant/utils/memory_repr_pytorch.py

```diff
@@ -0,0 +1,40 @@
+import cv2
+import torch
+
+from savant.utils.memory_repr import OpenCVGpuMatCudaArrayInterface, \
+    _numpy_type_to_opencv_mat_type
+
+
+__all__ = [
+    'opencv_gpu_mat_as_pytorch',
+    'pytorch_tensor_as_opencv_gpu_mat',
+]
+
+
+def opencv_gpu_mat_as_pytorch(gpu_mat: cv2.cuda.GpuMat) -> torch.Tensor:
+    """Returns PyTorch tensor for specified OpenCV GpuMat."""
+    torch_tensor = torch.tensor(OpenCVGpuMatCudaArrayInterface(gpu_mat), device="cuda")
+    if torch_tensor.dim() == 2:
+        return torch_tensor
+    return torch_tensor.permute(2, 0, 1)
+
+
+def pytorch_tensor_as_opencv_gpu_mat(tensor: torch.Tensor) -> cv2.cuda_GpuMat:
+    """Returns OpenCV GpuMat for specified PyTorch tensor.
+    Supports 2 and 3 dims arrays in CHW format for shape and `channel_first`
```

Comment:
I think there should be a note about the format. This case is different from cupy. Cupy ndimage has the same format as opencv, so we only mention the fact that the user should have the "expected" (for opencv) format. Here the situation is reversed. Torch has a different format, and you require the torch format, not opencv.

Reply:
I've made some changes to the conversions. Storage format in memory is now not important. I've removed this requirement from the description